run_xnat2bids.py

```diff
@@ -11,65 +13,115 @@
 
 
 logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.NOTSET)
+logging.getLogger('asyncio').setLevel(logging.WARNING)
```

Comment:
I think we talked about this, but it would be nice to change the level of logging via the `verbosity` input